prog/vnet/nic_nexus.rb

```diff
@@ -79,15 +80,15 @@ def before_run
       decr_start_rekey
       hop_wait_rekey_outbound_trigger
     end
-    donate
+    nap 1
```

Comment:
Why? if we are waiting for children to finish, shouldn't we donate time to them?

This might be okay if we are respirate which runs all Strands but will be difficult to deal with if we want to run a CI which needs to run a subset of the strands rooted at one strand.

Reply:
Yes, you're right. I was simply trying to see what happens if I progressed the children myself in dev mode. I changed all donate/nap calls properly